packages/std/src/entry_points.rs

```diff
@@ -0,0 +1,137 @@
+/// This macro generates the boilerplate required to call into the
+/// contract-specific logic from the entry-points to the WASM module.
+///
+/// It macro should be invoked in a global scope, and the argument to the macro
+/// should be the name of a rust module that is imported in the invocation scope.
+/// The module should export three functions with the following signatures:
+/// ```
+/// # use cosmwasm_std::{
+/// #     Storage, Api, Querier, Extern, Env, StdResult, Binary,
+/// #     InitResult, HandleResult, QueryResult,
+/// # };
+/// #
+/// # type InitMsg = ();
```

Comment:
What do all those # do here?

Reply:
check out `cargo doc --open`, and open the documentation for the macro :)
tl;dr they make them not show up in the docs.

Reply:
Those lines are required to make the example compile, but distract from the point I was trying to make.